datacleaner/datacleaner.py

```diff
@@ -72,11 +83,18 @@ def autoclean(input_dataframe, drop_nans=False, copy=False):
 
         # Encode all strings with numerical equivalents
         if str(input_dataframe[column].values.dtype) == 'object':
-            input_dataframe[column] = LabelEncoder().fit_transform(input_dataframe[column].values)
+            if encoder is None:
+                input_dataframe[column] = encoder().fit_transform(input_dataframe[column].values)
```

Comment:
Agreed on the kwargs, and I can do col-by-col encoding, but that will break support for HashingEncoder, which is multi-column.  Let me know if that's tolerable otherwise I can try to accommodate both somehow.